ios/FluentUI/Vnext/Divider/MSFDivider.swift

```diff
@@ -38,23 +30,8 @@ import SwiftUI
     @objc public init(orientation: MSFDividerOrientation,
                       spacing: MSFDividerSpacing,
                       theme: FluentUIStyle?) {
```

Comment:
This initializer can be merged with its convenience init; `FluentUIStyle` is part of the old theming system and should no longer be referenced in Fluent 2.0 controls.